server/authz/route_map.go

```diff
@@ -38,34 +39,45 @@ func newRouteInfo(object string, requiresAuthZ bool) *RouteInfo {
 	}
 }
 
+// GetBaseHref returns the base href of the server with a trailing slash.
+func GetBaseHref() string {
+	baseHref := sharedutil.LookupEnvStringOr("NUMAFLOW_SERVER_BASE_HREF", "/")
```

Comment:
This is not right. Even though we can use the the configmap to get the baseHref config, but it's passed through the arguments. We need to refactor the code to use the passed in argument.

Reply:
updated